src/components/learner-credit-management/data/hooks/useOfferSummary.test.js

```diff
@@ -0,0 +1,67 @@
+import { renderHook } from '@testing-library/react-hooks/dom';
```

Comment:
[context] Contents are unchanged from original tests. Simply moving tests for `useOfferSummary` into its own standalone test file, so we don't have a giant `hooks.test.js` file over time.